core/templates/dev/head/components/sharing_links_directive.html

```diff
@@ -0,0 +1,45 @@
+<script type="text/ng-template" id="components/sharingLinks">
+  <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
+    {% if SHARING_OPTIONS['gplus'] %}
+    <li>
```

Comment:
Indent this block by 2, similar to what was done before. Ditto below. The idea here is to optimize for developers reading the code, not for developers reading the final HTML -- the former happens more often.